es6-shim.js

```diff
@@ -1939,9 +1939,7 @@
 
     sign: function sign(value) {
       var number = Number(value);
-      if (number === 0) { return number; }
-      if (Number.isNaN(number)) { return number; }
-      return number < 0 ? -1 : 1;
+      return number < 0 ? -1 : number > 0 ? 1 : number;
```

Comment:
here i kind of prefer the explicit steps - it makes it very clear what the edge cases are for `0` and `NaN`.